README.md

```diff
@@ -1 +1,98 @@
-# Things3 Low-Level Python API to the Database
\ No newline at end of file
+Get your Things app data into a Python dict.
+
+## Install
+```python
+$ pip install things.py
+```
+
+## Run
+```pycon
```

Comment:
pylon -> python

Reply:
"pycon" here because we're in the Python console.  The regular Python highlighting makes some odd decisions.